test/unit/index.spec.js

```diff
@@ -0,0 +1,9 @@
+var cfg = require('../../lib/config')
+
+describe('index', () => {
+  var index = require('../../lib/index')
+
+  it('should expose the `config` object', () => {
+    expect(index.config).toBe(cfg)
```

Comment:
Thanks! :cold_sweat:

I couldn't run the tests locally because of out-dated dependencies and had now way to update them in the train. I will update this PR